lib/transport/net-transport.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var AbstractTransport = require('./abstract-transport.js');
+var net = require('net');
+var debug = require('debug')('amqp10:connection:nettransport');
+
+var NetTransport = function () {
+	AbstractTransport.call(this);
+	this._transport = null;
+};
+
+NetTransport.prototype = Object.create(AbstractTransport.prototype);
+NetTransport.prototype.constructor = NetTransport;
+
+NetTransport.prototype.register = function (client) {
+	client.registerProtocolHandler('amqp', this);
+};
+
+NetTransport.prototype.connect = function (address) {
+		debug('Connecting to ' + address.host + ':' + address.port + ' via straight-up sockets');
+		this._transport = net.connect({ port: address.port, host: address.host });
```

Comment:
another nitpick: this class itself represents the transport, however what's returned from `net.connect` (and likely in all cases) is a `socket`, so maybe `this._socket`